spec/features/publish/viewing_a_course_spec.rb

```diff
@@ -23,9 +24,42 @@
       given_i_am_authenticated_as_a_provider_user(course: build(:course, enrichments: [course_enrichment], funding_type: "salary"))
       when_i_visit_the_course_page
       then_i_should_see_the_description_of_the_salary_course
+      and_i_should_see_the_status_sidebar
     end
   end
 
+  describe "with a published and running course" do
+    scenario "i can view the published partial" do
+      given_i_am_authenticated_as_a_provider_user(course: build(:course, enrichments: [course_enrichment], funding_type: "salary", site_statuses: [build(:site_status, :findable)]))
+      when_i_visit_the_course_page
+      then_i_should_see_the_description_of_the_salary_course
+      and_i_should_see_the_status_sidebar
+      and_i_should_see_the_published_partial
+    end
+  end
+
+  describe "with a published with unpublished changes course" do
+    scenario "i can view the unpublished partial" do
+      given_i_am_authenticated_as_a_provider_user(course: build(:course, enrichments: [course_enrichment_unpublished_changes], funding_type: "salary"))
+      when_i_visit_the_course_page
+      then_i_should_see_the_description_of_the_unpublished_changes_course
+      and_i_should_see_the_status_sidebar
+      and_i_should_see_the_unpublished_partial
+    end
+  end
+
+  def and_i_should_see_the_status_sidebar
+    expect(publish_provider_courses_show_page).to have_status_sidebar
+  end
+
+  def and_i_should_see_the_unpublished_partial
+    expect(publish_provider_courses_show_page).to have_unpublished_partial
+  end
+
+  def and_i_should_see_the_published_partial
+    expect(publish_provider_courses_show_page).to have_published_partial
+  end
```

Comment:
For completeness can you do the withdrawn partial as well please

Reply:
I don't think I can do that yet, the `withdrawn` functionality is another ticket?